rpcd/playbooks/roles/elasticsearch/tasks/upgrade_pfc.yml

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
I'd rather name the file something better than "upgrade_pfc". Perhaps "pre_upgrade_checks"? "PFC" is kind of ... not very future maintainer friendly